crt_portal/cts_forms/views.py

```diff
@@ -232,6 +234,41 @@ def show_location_form_condition(wizard):
 
 class CRTReportWizard(SessionWizardView):
     """Once all the sub-forms are submitted this class will clean data and save."""
+
+    # overriding the get form to add checks to the hidden field and avoid 500s
```

Comment:
This does seem slightly heavy-handed, but I couldn't find a cleaner way to add a check to the step variable early in the process.